.gitignore

```diff
@@ -27,7 +28,6 @@ docs/_build/
 # Covers JetBrains PyCharm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 # User-specific stuff
-.idea/**/workspace.xml
```

Comment:
I'd rather keep the workspeace.xml out of the gitrepo

Reply:
Yes! I removed the .idea directory, probably that's why I did this. Sorry about that.

Reply:
No worries, I've found another instance of this in the gitingore file. All good!